src/components/SingleReview.jsx

```diff
@@ -18,6 +18,25 @@ const Review = () => {
         })
     }, [review_id])
 
+    const voteOnReview = () => {

```

Comment:
On line 52, you are passing in `review.review_id` as the argument into the function of the <button> but on line 21 here you aren't taking any parameters, i'd probably have `review_id` as an argument here, and then on line 30... pass that `review_id` into the axios request instead of `review.review_id` - that way you'll be certain the correct review id is being passed in.